gulpfile.js

```diff
@@ -465,7 +468,21 @@ function deployCesium(bucketName, uploadDirectory, cacheControl) {
         })
         .then(function() {
             console.log('Cleaned ' + existingBlobs.length + ' files.');
+        })
+        .catch(function(error) {
```

Comment:
There's no reason for the catch here because it will bubble up to the parent promise.